packages/components/src/color-palette/stories/index.js

```diff
@@ -0,0 +1,53 @@
+/**
+ * External dependencies
+ */
+import { object } from '@storybook/addon-knobs';
+
+/**
+ * WordPress dependencies
+ */
+import { useState } from '@wordpress/element';
+
+/**
+ * Internal dependencies
+ */
+import ColorPalette from '../';
+
+export default { title: 'Color Palette', component: ColorPalette };
+
+export const _default = () => {
+	const [ color, setColor ] = useState( '#f00' );
```

Comment:
We can't use hooks in stories. Storyshots fail. See #18030.

Reply:
Fixed.